test/drag/drag.js

```diff
@@ -1,5 +1,5 @@
-//load a lot of stuff ...
-steal(function(){
+steal("../../browser/jquery", "../../node_modules/jquerypp/index", function($){
+	window.jQuery = $;
```

Comment:
It's not needed.  There was some copy/paste/debugging going on.  I'll remove it.